--- src/drivers/usb/linux/linux-usb.c	2008-12-20 14:13:10.000000000 +0100
+++ src/drivers/usb/linux/linux-usb.c.oden	2008-12-20 14:13:26.000000000 +0100
@@ -145,7 +145,7 @@ static int open_usb_device(UPSINFO *ups)
    char devname[MAXSTRING];
    USB_DATA *my_data = (USB_DATA *)ups->driver_internal_data;
    const char *hiddev[] =
-      { "/dev/usb/hiddev", "/dev/usb/hid/hiddev", "/dev/hiddev", NULL };
+      { "/dev/hiddev", "/dev/usb/hiddev", "/dev/usb/hid/hiddev", NULL };
    int i, j, k;
 
    /*
