--- src/Makefile.link	2011-05-03 09:17:05.000000000 +0200
+++ src/Makefile	2011-05-03 09:17:21.000000000 +0200
@@ -26,10 +26,10 @@
 all-targets: apcupsd apcaccess apctest smtp 
 
 apcupsd: $(common_obj) $(apcupsd_obj) $(APCDRVLIBS) $(APCLIBS)
-	$(LINK) $(DRVLIBS)
+	$(LINK) $(DRVLIBS) -lstdc++
 
 apctest: $(common_obj) $(apctest_obj) $(APCDRVLIBS) $(APCLIBS)
-	$(LINK) $(DRVLIBS)
+	$(LINK) $(DRVLIBS) -lstdc++
 
 apcaccess: $(apcaccess_obj) $(APCLIBS)
 	$(LINK)
