--- apcupsd-3.10.16/platforms/mandrake/apcupsd.in~	2004-12-30 09:18:03.882985839 +0100
+++ apcupsd-3.10.16/platforms/mandrake/apcupsd.in	2004-12-30 09:18:03.882985839 +0100
@@ -24,6 +24,9 @@
 # check for stale lockfiles
        LOCKDIR=`grep "^LOCKFILE" @sysconfdir@/apcupsd.conf | awk '{ print $2}'`
        DEVICE=`grep "^DEVICE" @sysconfdir@/apcupsd.conf | awk '{ print $2 }' | sed -e 's@/dev/@@g'`
+       if [ -z "${DEVICE}" ]; then
+          DEVICE=usb
+       fi
        LOCKFILE="${LOCKDIR}/LCK..${DEVICE}"
        if [ -f $LOCKFILE ]; then
 	  PID=`cat $LOCKFILE | awk '{ printf("%d",$1) }'`
